### DISCUSSION TITLE
Add a row to a table

### DISCUSSION THREAD

**Discussion by femo1de** [ShapeCrawler/ShapeCrawler#307] [Ideas]
Is there a way to manipulate the rows in a table, for example add a new row?

**Comment by AdamShakhabov** [comment]
There is only a couple of APIs to update table, for example:

```c#
using var pres = SCPresentation.Open(@"table.pptx", true);
var table = pres.Slides[0].Shapes.GetByName<ITable>("Table 1");

// remove row by index
table.Rows.RemoveAt(0);
```

You can find more samples [**here**](https://github.com/ShapeCrawler/ShapeCrawler/wiki/Examples#table). Adding a new row is not supported yet.

**Comment by AdamShakhabov** [comment]
The new `IRowCollection.Add()` method has been added and will be available in the next v0.39.0 release.
